src/client/particles.cpp

```diff
@@ -38,9 +39,10 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 v3f random_v3f(v3f min, v3f max)
 {
-	return v3f( rand()/(float)RAND_MAX*(max.X-min.X)+min.X,
-			rand()/(float)RAND_MAX*(max.Y-min.Y)+min.Y,
-			rand()/(float)RAND_MAX*(max.Z-min.Z)+min.Z);
+	return v3f(
+		rand() / (float)RAND_MAX * (max.X - min.X) + min.X,
+		rand() / (float)RAND_MAX * (max.Y - min.Y) + min.Y,
+		rand() / (float)RAND_MAX * (max.Z - min.Z) + min.Z);
```

Comment:
As you touch this code, I want to note that, according to `numeric.h`, "VC++ rand() sucks".
https://github.com/minetest/minetest/blob/b2065756cf9c90d493846a559587e4acec4a7851/src/util/numeric.h#L214

Reply:
Not _quite_ related to this PR, but interesting. What do you suggest I do here?

Reply:
I could replace `rand` with `myrand` (PRNG implementation). But I prefer to do it in a different PR, and keep this one clean - it has already got 2 code-style commits apart from the bugfix.